contracts/HubPool.sol

```diff
@@ -144,12 +158,23 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
     ) Testable(_timer) {
         lpTokenFactory = _lpTokenFactory;
         finder = _finder;
+        protocolFeeCaptureAddress = owner();
     }
 
     /*************************************************
      *                ADMIN FUNCTIONS                *
      *************************************************/
 
+    function setProtocolFeeCaptureAddress(address newProtocolFeeCaptureAddress) public onlyOwner {
```

Comment:
Should this function logic be collapsed with the following one?

Reply:
ye, good point. can do how we do the `setBond` method.